### ISSUE TITLE
I can't take a screenshot by pressing the hotkey

### ISSUE THREAD

**Issue by Toadfield** [flameshot-org/flameshot#1577]
**Flameshot version**
0.9.0
**Describe the bug**
I can't take a screenshot by pressing the hotkey,if I press the hotkey,then nothing happens.
Btw I can take a screenshot by launching flameshot manually.
**To Reproduce**
Go into kde shortcut settings,search for "flameshot" and then enable a shortcut and press it.
**Expected behavior**
That I can take a screenshot by pressing the shortcuts.
**System Information**
![image](https://user-images.githubusercontent.com/68649672/115883902-99393680-a44e-11eb-80b1-a5dff597f577.png)

**Comment by twocountriestrus** [comment]
Gracias 


Enviado desde Yahoo Mail para iPhone


El viernes, abril 23, 2021, 10:13 a. m., Toadfield ***@***.***> escribió:




Flameshot version
0.9.0
Describe the bug
I can't take a screenshot by pressing the hotkey,if I press the hotkey,then nothing happens.
Btw I can take a screenshot by launching flameshot manually.
To Reproduce
Go into kde shortcut settings,search for "flameshot" and then enable a shortcut and press it.
Expected behavior
That I can take a screenshot by pressing the shortcuts.
System Information

**Comment by mmahmoudian** [comment]
Your explanation is confusing to me. Do you mean:
1. when you press <kbd>Printscreen</kbd> button, the flameshot does not open
or
2. when you have opened flameshot and are annotating the screenshot, you cannot press a key to capture it (e.g <kbd>Ctrl</kbd>+<kbd>c</kbd>?

**Comment by Toadfield** [comment]
> when you press Printscreen button, the flameshot does not open

That is what I mean.

**Comment by mmahmoudian** [comment]
That is the part that is not handled by Flameshot. Please follow instructions here:

https://flameshot.org/key-bindings/#on-kde-plasma-desktop

**Comment by Toadfield** [comment]
That is what I did,I added that and then there is a flameshot entry and I set a hotkey,but then if I press the hotkey it doesn't work.

**Comment by mmahmoudian** [comment]
Can you post some screenshots of your KDE shortcuts settings. Specifically the one you set for Flameshot. I'm interested in the command section of the shortcut

**Comment by sr1canskhsia** [comment]
For anyone who encountered the same problem: make sure you have the package `qt5-tools` installed!
Somehow this package is only listed as a make dependency on arch...

**Comment by mmahmoudian** [comment]
@sr1canskhsia What is your operating system? As far as I know, the instructions are correct:
https://flameshot.org/guide/installation/compilation/

**Comment by sr1canskhsia** [comment]
@mmahmoudian I'm on Arch Linux with KDE Plasma. I installed flameshot 0.9.0 from the Arch official repo. After that I removed all the make dependencies with `pacman -Qttdq | pacman -Rns -`
I know that `qt5-tools` is required to build flameshot. But if it is also needed to make KDE shortcuts work properly, then the documentation should probably mention that. Or we should probably contact the Arch package maintainer to add `qt5-tools` as an optional dependency.
